Yoast/yoast-components
# Fix promise for jQuery requests

The postJSON helper always failed when using the jQuery post function. I modified the promise to make it work again.

Testing:
1. Enable jQuery
- npm install jquery
- in postJSON.js add `import jQuery from "jquery";` on the top of the file.
  1. Start the application and go through the steps.
  2. Check if jQuery is used and the steps load.

*To check if the wizard is actually using jquery add a console log before the return 

postJSONjQuery.
Fixes #22
